corporate/migrations/0011_move_exempt_from_from_license_number_check_to_customer_model.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 3.2.4 on 2021-06-18 18:39
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    """
+    We haven't set the values for this field for the relevant organizations
+    as of this moment, so we can simply drop the column from CustomerPlan
+    and add it to Customer without worrying about losing the values.
+    """
+
+    dependencies = [
+        ("corporate", "0010_customerplan_exempt_from_from_license_number_check"),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name="customerplan",
+            name="exempt_from_from_license_number_check",
+        ),
+        migrations.AddField(
```

Comment:
Don't we need a migration to copy data from the one location to the other?

Reply:
@timabbott  I wrote a brief justification in the migration docstring - we haven't set this value for anyone yet, so it should be fine to skip it. Unless we don't intend to deploy this for a longish time.

Reply:
Ahh, OK, makes sense.